concrete/attributes/boolean/controller.php

```diff
@@ -200,4 +187,69 @@ public function getAttributeKeySettingsClass()
         return BooleanSettings::class;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @see \Concrete\Core\Attribute\SimpleTextExportableAttributeInterface::getAttributeValueTextRepresentation()
+     */
+    public function getAttributeValueTextRepresentation()
+    {
+        $value = $this->getAttributeValueObject();
+        if ($value === null || $value->getValue() === null) {
+            $result = '';
+        } else {
+            $result = $value->getValue() ? '1' : '0';
+        }
+
+        return $result;
+	}
+
+	/**
+	 * {@inheritdoc}
+	 *
+	 * @see \Concrete\Core\Attribute\SimpleTextExportableAttributeInterface::updateAttributeValueFromTextRepresentation()
+	 */
+	public function updateAttributeValueFromTextRepresentation($textRepresentation, ErrorList $warnings)
+    {
+	    $value = $this->getAttributeValueObject();
```

Comment:
Couldn't we just use `$value->setValue(filter_var($textRepresentation, FILTER_VALIDATE_BOOLEAN))`?

Reply:
Why not! I'll update this